src/Workspaces/Core/Portable/SolutionCrawler/IIncrementalAnalyzer.cs

```diff
@@ -30,5 +30,10 @@ internal interface IIncrementalAnalyzer
         Task RemoveProjectAsync(ProjectId projectId, CancellationToken cancellationToken);
 
         bool NeedsReanalysisOnOptionChanged(object sender, OptionChangedEventArgs e);
+
+        /// <summary>
+        /// Flag indicating if the analysis result for documents are dependent on whether or not it is an active document.
```

Comment:
can you give an example?

Reply:
actually an example of both true/false values would be good.

Reply:
Code is now removed.